cmd/virt-handler/virt_launcher.cil

```diff
@@ -53,4 +53,8 @@
     (allow process nfs_t (dir (mounton)))
     (allow process proc_t (dir (mounton)))
     (allow process proc_t (filesystem (mount unmount)))
+    ;
+    ; This is needed for passt.
+    ; The policy will be removed from here once it will be installed via the passt package.
+    (allow process tmpfs_t (filesystem (mount)))
```

Comment:
@AlonaKaplan sorry for commenting on a closed PR, but I ran into this while working on https://github.com/kubevirt/kubevirt/pull/8402.
In that PR, I run all VMIs (except for hugepages/virtiofs ones) under container_t, so without this permission, and it surprisingly doesn't break passt tests.
Is that permission not needed anymore?
Thanks!